website/templates/labels/png5/conf.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Beautiful one

I like the QRCode on both sides :)

add a comment 
// Modern :: Wallson 2 (with QR)

Reply:
done

Reply:
I also think it's really nice picture, wonder why this image was just laying around for quite some time without usage =)